gatsby/content/blog/2020/12/2020-12-07-gitter-now-speaks-matrix.mdx

```diff
@@ -12,8 +12,8 @@ Hi all,
 It’s been just over 2 months since we revealed that [Gitter was going to join Matrix](https://matrix.org/blog/2020/09/30/welcoming-gitter-to-matrix) - and
 we are incredibly proud to announce that Gitter has officially turned on true native Matrix connectivity:
 **all public Gitter rooms are now available natively via Matrix, and all Gitter users now natively exist on Matrix.**
-So, if you wanted to join the official Scala language support room at [https://gitter.im/scala/scala](https://gitter.im/scala/scala)
-from Matrix, just head over to [#scala_scala:gitter.im](https://matrix.to/#/#scala_scala:gitter.im) and \*boom\*, you’re in!
+So, if you wanted to join the official Scala language support room at [https://gitter.im/scala/center](https://gitter.im/scala/center)
+from Matrix, just head over to [#scala_cente:gitter.im](https://matrix.to/#/#scala_center:gitter.im) and \*boom\*, you’re in!
```

Comment:
Typo in `#scala_cente:gitter.im` -> `#scala_center:gitter.im`

Reply:
How embarrassing; thanks for correcting!

Reply:
@habi To clarify, I didn't add a fix to this PR. Can you add another commit or rebase to fix this spot?

I think we're good to merge after.

Reply:
I did this online on GitHub, so I can only add another commit.

Reply:
Thanks for the update @habi! To make sure everything works, I joined `#scala_center:gitter.im` and discovered that there isn't much activity going on in that room and wouldn't be a good representation for people just stopping by.

`#nodejs_node:gitter.im` (https://gitter.im/nodejs/node) seems like a good alternative. Sorry for the extra iteration here 😓

Reply:
> Sorry for the extra iteration here 😓

No worries!